src/Modal.js

```diff
@@ -340,7 +365,7 @@ const Modal = React.createClass({
     let doc = ownerDocument(this);
     let container = getContainer(this.props.container, doc.body);
 
-    modalManager.add(this, container, this.props.containerClassName);
```

Comment:
same thing, I just moved the default instance to a defaultProp so you can configure them via the component